client/src/modules/stock/adjustment/adjustment.js

```diff
@@ -227,9 +227,11 @@ function StockAdjustmentController(
 
     movement.lots = lots;
 
+    vm.$loading = true;
```

Comment:
A better solution than this is to change [this line](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/stock/adjustment/adjustment.html#L143) from
```html
<bh-loading-button loading-state="StockCtrl.$loading" disabled="!StockCtrl.validForSubmit">
```
to
```html
<bh-loading-button loading-state="StockForm.$loading" disabled="!StockCtrl.validForSubmit">
```

Basically, the author meant to type "StockForm", but they typed "StockCtrl" originally.  This should fix it without introducing new code in the controller.